packages/react-flex/Spec.md

```diff
@@ -0,0 +1,540 @@
+| `shrink` | boolean \| number \| string = "`inherit`", "`initial`", "`unset`"                               | `1`           | Abstraction of `flex-shrink` and override of Flex's `shrink`. A `true` value will translate into `1` for ease of use and retro-compatibility with Stack. |
+| `push`   | boolean                                                                                         | `false`       | Defines an auto margin depending on the flex direction. Kept for retro-compatibility but perhaps can be removed.                                         |
+| `basis`  | string (_see [flex-basis](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-basis#values)_) | "`auto`"      | For clarity, renaming of v0 Flex's `size` prop to `basis`.                                                                                               |
+| `order`  | number                                                                                          | `0`           | Abstraction of `order`.                                                                                                                                  |
+
+#### _Deprecating_
+
+| Original Component | Name            | Comments                                               |
+| ------------------ | --------------- | ------------------------------------------------------ |
+| StackItem          | `disableShrink` | Extraneous prop given the existance of `shrink`.       |
+| StackItem          | `verticalFill`  | Redundant. Same as a style override of `height: 100%`. |
+| FlexItem           | `size`          | Renamed to `basis`.                                    |
+
+## Sample Code
+
+### Basic
+
+```HTML
+<Flex>
+ <span>1</span>
+ <span>2</span>
+</Flex>
+```
+
+### Direction
+
+```HTML
+<Flex direction="column-reverse">
+  <span>bottom</span>
+  <span>top</span>
+</Flex>
+```
+
+### Alignment
+
+```HTML
+<Flex verticalAlign="center">
+  <span>vertically centered</span>
+  <span>vertically centered</span>
+</Flex>
+```
+
+```HTML
+<Flex horizontalAlign="center">
+  <span>horizontally centered</span>
+  <span>horizontally centered</span>
+</Flex>
+```
+
+### Spacing
+
+```HTML
+<Flex gap="5rem">
+  <span>5rem around me</span>
+  <span>5rem around me</span>
+</Flex>
+```
+
+### Wrapping
+
+```HTML
+<Flex wrap>
+  <span>1</span>
+  <span>2</span>
+</Flex>
+```
+
+### Order
+
+```HTML
+<Flex>
+  <span>2</span>
+  <span order={1}>1</span>
+</Flex>
+```
+
+### Growing
+
+#### Globally
+
+```HTML
+<Flex grow>
+  <span>wide</span>
+  <span>wide</span>
+</Flex>
+```
+
+#### Individually
+
+```HTML
+<Flex>
+  <span>thin</span>
+  <span grow>wide</span>
+</Flex>
+```
+
+### Shrinking
+
+#### Globally
+
+```HTML
+<Flex shrink={false}>
+  <span>wide</span>
+  <span>wide</span>
+</Flex>
+```
+
+#### Individually
+
+```HTML
+<Flex>
+  <span>thin</span>
+  <span shrink={false}>wide</span>
```

Comment:
a boolean value will not work since React will actually throw console errors at you and advise making these string properties. Codesandbox example: https://codesandbox.io/s/adoring-forest-40vuf

Reply:
I guess we can just avoid this altogether by keeping the same pattern of using `<Flex.Item>`. I'll update the spec